python/nano/src/bigdl/nano/utils/common/checker.py

```diff
@@ -62,3 +67,14 @@ def _avx512_checker():
     '''
     cpuinfo = CPUInfo()
     return cpuinfo.has_avx512
+
+
+def _avx2_checker():
```

Comment:
This checker is used to avoid avx2 checking issue involved by IPEX (since AVX2 is involved by [Haswell](https://en.wikipedia.org/wiki/Haswell_(microarchitecture)) processor in 2013, very few users are using previous products). Should we post an issue to IPEX team?

Reply:
Yeah, I will raise a related issue to IPEX team. But we still need this checker to avoid avx2 checking issue as least for ipex2.0 .